ui/src/helpers/api.js

```diff
@@ -76,4 +76,12 @@ export function questionsQuery() {
   return request
     .get('/server/questions')
     .set('Content-Type', 'application/json');
+}
+
+export function saveQuestions(record) {
```

Comment:
Good question!  Just clarify, we could call it `variableNamesOnlyMatterToHumans`:)  But clarity in naming and documentation is usually particularly important when crossing system boundaries like here (eg., from the UI in the browser to the server code).